packages/terra-folder-tree/src/subcomponents/FolderTreeItem.module.scss

```diff
@@ -0,0 +1,30 @@
+@import '../clinical-lowlight-theme/FolderTree.module';
+@import '../orion-fusion-theme/FolderTree.module';
+
+:local {
+  .folder-tree-item {
+    align-items: center;
+    background-color: var(--terra-folder-tree-item-background-color, #fff);
```

Comment:
I think it is better to use background instead of background-color.  The background style would allow for both colors and gradients without a future style change.